edx_exams/apps/core/templates/email/proctoring_attempt_submitted.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+
+<p>
+    {% block introduction %}
+    {% blocktrans %}
+        Hello {{ username }},
+    {% endblocktrans %}
+    {% endblock %}
+</p>
+<p>
+    {% block status_information %}
+    {% blocktrans %}
+        Your proctored exam "{{ exam_name }}" in
+        <a href="{{ course_url }}">{{ course_url }}</a> was submitted
```

Comment:
These used to have the course name here. For right now we don't have that in the data model.